pkg/namespace/cmd.go

```diff
@@ -106,11 +106,7 @@ func (c cmd) Modify(ns Namespace, b *Builder) error {
 		}
 		return ns.Import(host, remotepath, mountpoint, c.flag)
```

Comment:
This looks good. This one change is a case for making this into more than one PR. It's SO BIT, it's exhausting to read.